modules/project-structure/01-project-structure.qmd

```diff
@@ -0,0 +1,180 @@
+---
+title: Project structure
+subtitle: 'In this topic, you will learn about the project structure and how to manage your project in R properly.'
+title-block-banner: yes
+engine: knitr
+filters:
+  - webr-teachr
+execute: 
+  echo: false
+---
+
+```{r}
+knitr::opts_chunk$set(
+    fig.width = 8,
+    fig.height = 8,
+    fig.align = "center")
+```
+
+# Paths {#sec-path}
+
+The **paths** are used to identify the location of a file, computer, device, or web page. You could think of it as a direction pointing to the destination.
+
+## What does it look like?
+
+### File path
```

Comment:
I think the depth of these headers are the wrong way around - the higher order headers should be the broader category